### PR TITLE
cbellee - ACA Networking Bicep examples

### PR DESCRIPTION
**Pull request by cbellee**
Added example Bicep deployments for ACA - Networking session (Internal vNet, Azure Frontdoor integration)

### CODE REVIEW

File: content/containers/aca-networking/assets/internal-vnet/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# get deployment template outputs
+APP_FQDN=`az deployment group show --resource-group $RG_NAME --name 'infra-deployment' --query properties.outputs.appFqdn.value --output tsv`
+BASTION_HOST_NAME=`az deployment group show --resource-group $RG_NAME --name 'infra-deployment' --query properties.outputs.bastionHostName.value --output tsv`
+VM_ID=`az deployment group show --resource-group $RG_NAME --name 'infra-deployment' --query properties.outputs.vmId.value --output tsv`
+
+echo "APP_FQDN: https://$APP_FQDN"
+
+# open ssh tunnel to VM in the same vnet as the ACA environment
+az network bastion ssh \
```
Comment (willvelida) [inline content/containers/aca-networking/assets/internal-vnet/deploy.sh:29]: Could you add a comment with a command to install the ssh extension here with `az extension add --name ssh` please? I got an error when running it for the first time and adding the extension helped.

File: content/containers/aca-networking/assets/internal-vnet/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	--name $BASTION_HOST_NAME \
+	--resource-group $RG_NAME \
+	--target-resource-id $VM_ID \
+	--auth-type ssh-key \
+	--username 'localadmin' \
+	--ssh-key '~/.ssh/id_rsa'
+
+az network bastion tunnel --name $BASTION_HOST_NAME --resource-group $RG_NAME --target-resource-id $VM_ID --resource-port 22 --port 50022
```
Comment (willvelida) [inline content/containers/aca-networking/assets/internal-vnet/deploy.sh:37]: Does this command need to happen before we ssh into the VM?

**Review by willvelida — CHANGES_REQUESTED** [review]

File: content/containers/aca-networking/assets/internal-vnet/deploy.sh (hunk 1 above)
Comment (cbellee) [reply content/containers/aca-networking/assets/internal-vnet/deploy.sh:29]: Good point, I'll add this now. Thanks!

File: content/containers/aca-networking/assets/internal-vnet/deploy.sh (hunk 2 above)
Comment (cbellee) [reply content/containers/aca-networking/assets/internal-vnet/deploy.sh:37]: Yes, the command opens an ssh tunnel through Azure Bastion to the VM. You then SSH to localhost on the port specified (50022 in this case).

**Review by willvelida — DISMISSED** [review]

**Review by willvelida — APPROVED** [review]

**Pull request merged**